packages/lodestar/src/sync/gossip/handler.ts

```diff
@@ -117,10 +117,11 @@ export class BeaconGossipHandler {
     signature: altair.SyncCommitteeSignature
   ): Promise<void> => {
     // TODO: Review if we need to check shouldProcessAttestation() like with onAttestation
-
-    // TODO: Do this much better to be able to access this property in the handler
-    const indexInSubCommittee = (signature as SyncCommitteeSignatureIndexed).indexInSubCommittee;
-    this.db.syncCommitee.add(subnet, signature, indexInSubCommittee);
+    if (this.network.syncnetsService.shouldProcess(subnet, signature.slot)) {
```

Comment:
I think we don't need this check, we should always process

Reply:
Sounds good, removing check